packages/react-components/react-drawer/stories/src/Drawer/DrawerResizable.stories.tsx

```diff
@@ -106,27 +121,78 @@ export const Resizable = () => {
     };
   }, [resize, stopResizing]);
 
+  const [resizeInput, setResizeInput] = React.useState<string>(sidebarWidth.toString());
+
+  const onResizeInputChange: InputProps['onChange'] = (ev, data) => {
+    if (data.value) {
+      setResizeInput(data.value);
+    }
+  };
+
   return (
-    <div className={mergeClasses(styles.root, isResizing && styles.rootResizerActive)}>
-      <div className={styles.container}>
-        <InlineDrawer
-          open
-          ref={sidebarRef}
-          className={styles.drawer}
-          style={{ width: `${sidebarWidth}px` }}
-          onMouseDown={e => e.preventDefault()}
-        >
-          <DrawerHeader>
-            <DrawerHeaderTitle>Default Drawer</DrawerHeaderTitle>
-          </DrawerHeader>
-          <DrawerBody>
-            <p>Resizable content</p>
-          </DrawerBody>
-        </InlineDrawer>
-        <ResizeComponent />
+    <>
+      <div className={mergeClasses(styles.root, isResizing && styles.rootResizerActive)}>
+        <div className={styles.container}>
+          <InlineDrawer
+            open
+            ref={sidebarRef}
+            className={styles.drawer}
+            style={{ width: `${sidebarWidth}px` }}
+            onMouseDown={e => e.preventDefault()}
+          >
+            <DrawerHeader>
+              <DrawerHeaderTitle>Default Drawer</DrawerHeaderTitle>
+            </DrawerHeader>
+            <DrawerBody>
+              <p>Resizable content</p>
+            </DrawerBody>
+          </InlineDrawer>
+          <Dialog>
+            <DialogTrigger disableButtonEnhancement>
+              <div
+                className={mergeClasses(styles.resizer, isResizing && styles.resizerActive)}
+                onMouseDown={startResizing}
+              />
+            </DialogTrigger>
+            <DialogSurface>
+              <DialogBody>
+                <DialogTitle
+                  action={
+                    <DialogTrigger action="close">
+                      <Button appearance="subtle" aria-label="close" icon={<Dismiss20Regular />} />
+                    </DialogTrigger>
+                  }
+                >
+                  Resize drawer
+                </DialogTitle>
+                <DialogContent>
+                  <div className={styles.dialogContent}>
+                    <Label htmlFor={inputId}>Enter desired drawer width pixels:</Label>
+                    <Input id={inputId} onChange={onResizeInputChange} defaultValue={resizeInput} />
```

Comment:
Since this input is supposed to accept number values, it would makes sense to set it as `type="number"`